tests/concepts/test_DualView.cpp

```diff
@@ -0,0 +1,30 @@
+#include "gtest/gtest.h"
+
+#include "kokkos-utils/concepts/DualView.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+/**
+ * @file
+ *
+ * @addtogroup unittests
+ *
+ * **Concepts related to @c Kokkos::DualView**
+ *
+ * This group of tests check the behavior of our concepts related to @c Kokkos::DualView.
+ */
+
+namespace Kokkos::utils::tests::concepts
+{
+
+//! @test Check that @ref Kokkos::utils::concepts::DualView works as expected.
+TEST(concepts, DualView)
+{
+    using dual_view_1d_t = Kokkos::DualView<int[5]   , execution_space>;
+    using dual_view_2d_t = Kokkos::DualView<int[5][5], execution_space>;
+
+    static_assert(Kokkos::utils::concepts::DualView<dual_view_1d_t>);
```

Comment:
It could be interesting to test briefly that the concepts can differentiate between view and dualview, e.g. checking whether a dual view satisfies the view concept should yield false.

Reply:
OK for you like this then ?

Reply:
Ok! That’s exactly what I was thinking.